relay-event-normalization/src/validation.rs

```diff
@@ -44,7 +44,9 @@ pub fn validate_transaction(
 
     validate_transaction_timestamps(event, config)?;
     validate_trace_context(event)?;
-    validate_spans(event)?;
+    // Transactions with spans with invalid timestamps are not expected to be
+    // rejected.
```

Comment:
This statement is not really true right now, since in the `validate_span` function we can return an error if either `start` or `end` timestamps are absent (which is kind of invalid too), so we will return the error from `validate_transaction` as well and reject the `transaction` as well.

It would be great to change this comment here to reflect what condition wouldn't actually reject the transaction.

Reply:
Thanks for pointing it out! I reworded it in [b9b8b70](https://github.com/getsentry/relay/pull/3013/commits/b9b8b70620e9d5f55331e3c87c108f694ef64d17), let me know if this is still unclear.